### PR TITLE
Add getting started guide for unmanaged cluster

### PR DESCRIPTION
**Pull request by joshrosso**
## What this PR does / why we need it

This commit introduces the getting started guide for unmanaged cluster.
This replaces (in the TOC) the existing standalone-cluster docs.

## Details for the Release Notes (PLEASE PROVIDE)

```release-note
Getting started guide introduced for unmanaged-clusters 
```

## Which issue(s) this PR fixes

Fixes: #TODO

## Describe testing done for PR

Run `hugo serve`

image

## Special notes for your reviewer

Run hugo locally and review the docs.

### CODE REVIEW

**Comment by joshrosso** [comment]
cc @kcoriordan for our conversation tomorrow around the unmanaged-cluster docs.

File: docs/site/content/docs/assets/prereq-unmanaged-linux.md
Comment (stmcginnis) [inline docs/site/content/docs/assets/prereq-unmanaged-linux.md:3]: It can probably deploy with 1 GB, but do we really want to state that as a minimum?

Comment (joshrosso) [reply docs/site/content/docs/assets/prereq-unmanaged-linux.md:3]: Do you mean 1 CPU? Minimum is `2 GB`.

Comment (stmcginnis) [reply docs/site/content/docs/assets/prereq-unmanaged-linux.md:3]: Yeah, distracted. I meant 1 CPU.

Comment (joshrosso) [reply docs/site/content/docs/assets/prereq-unmanaged-linux.md:3]: I think 1 CPU may be adequate. Do you have a machine (non-linux/docker desktop) that you could validate with?

Comment (stmcginnis) [reply docs/site/content/docs/assets/prereq-unmanaged-linux.md:3]: I could spin up a linux host, but otherwise no, I don't. But even linux it seems like it would be safer to have more than 1 CPU if they want to actually run anything on the cluster.

But let's leave this at 1 for now until we hear feedback that would make us want to change it.

Comment (joshrosso) [reply docs/site/content/docs/assets/prereq-unmanaged-linux.md:3]: i'll do a test with qemu

Comment (joshrosso) [reply docs/site/content/docs/assets/prereq-unmanaged-linux.md:3]: @stmcginnis quick profiling of unmanaged-cluster running with default config:

profiling-unmanaged

Since CPU is a compressible/scheduled resource, it had no issues during bootstrapping too (even though it hit 100 a few times)

File: docs/site/content/docs/latest/getting-started-unmanaged.md
Comment (stmcginnis) [inline docs/site/content/docs/latest/getting-started-unmanaged.md:1]: The hugo build errors appear to be from these `markdownlint-disable` lines. I think I remember hitting something similar when trying to put these directives at the top of include files.

It appears hugo and mdlint are both happy if you move them down to follow the H1 title.

**Comment by jpmcb** [comment]
Looks like some markdown nit pick linting errors. Otherwise, this is looking good to me!

**Review by jpmcb — CHANGES_REQUESTED** [review]
🛑 Hold this PR until we've released `unmanaged-cluster`: otherwise, the live docs site will reflect something that's not yet in a release

File: docs/site/content/docs/assets/install-package-to-cluster.md
Comment (kcoriordan) [inline docs/site/content/docs/assets/install-package-to-cluster.md:38]: contour should be cert-manger here?

File: docs/site/content/docs/latest/getting-started-unmanaged.md
Comment (joshrosso) [reply docs/site/content/docs/latest/getting-started-unmanaged.md:3]: addressed; thanks for figuring this out.

**Comment by joshrosso** [comment]
> Looks like some markdown nit pick linting errors. Otherwise, this is looking good to me!

addressed :+1:

File: docs/site/content/docs/assets/install-package-to-cluster.md
Comment (kcoriordan) [inline docs/site/content/docs/assets/install-package-to-cluster.md:12]:
```suggestion
    tkg-core-repository  projects.registry.vmware.com/tce/main                {{< pkg_repo_latest >}}                   Reconcile succeeded           tanzu-package-repo-global
```

File: docs/site/content/docs/latest/getting-started-unmanaged.md
Comment (kcoriordan) [inline docs/site/content/docs/latest/getting-started-unmanaged.md:85]:
```suggestion
* [Unmanaged Clusters Reference Documentation](/docs/latest/ref-unmanaged-cluster)
```

File: docs/site/content/docs/latest/ref-unmanaged-cluster.md
Comment (kcoriordan) [inline docs/site/content/docs/latest/ref-unmanaged-cluster.md:61]:
```suggestion
`unmanaged-cluster` takes configuration from the multiple sources in the
```

File: docs/site/content/docs/latest/ref-unmanaged-cluster.md
Comment (kcoriordan) [inline docs/site/content/docs/latest/ref-unmanaged-cluster.md:30]:
```suggestion
To create a cluster named, run:
```

File: docs/site/content/docs/latest/ref-unmanaged-cluster.md
Comment (kcoriordan) [inline docs/site/content/docs/latest/ref-unmanaged-cluster.md:64]: Missing content?

File: docs/site/content/docs/latest/ref-unmanaged-cluster.md
Comment (kcoriordan) [inline docs/site/content/docs/latest/ref-unmanaged-cluster.md:113]: Missing content?

File: docs/site/content/docs/latest/ref-unmanaged-cluster.md
Comment (kcoriordan) [inline docs/site/content/docs/latest/ref-unmanaged-cluster.md:115]: Missing content?

File: docs/site/content/docs/latest/ref-unmanaged-cluster.md
Comment (kcoriordan) [inline docs/site/content/docs/latest/ref-unmanaged-cluster.md:126]:
```suggestion
By design, `unmanaged-clusters` do not lifecycle-manage Kubernetes. They are not meant to be long-running with real workloads. To change Kubernetes versions, delete the existing cluster and create a new cluster with a different configuration.
```

File: docs/site/content/docs/latest/ref-unmanaged-cluster.md
Comment (kcoriordan) [inline docs/site/content/docs/latest/ref-unmanaged-cluster.md:5]:
```suggestion
[Getting Started with Unmanaged Clusters](getting-started-unmanaged/.
```

File: docs/site/content/docs/assets/install-package-to-cluster.md
Comment (joshrosso) [reply docs/site/content/docs/assets/install-package-to-cluster.md:12]: @kcoriordan, unfortunately the newest package repo is not necessarily what unmanaged is going to ship with, in fact, it'll ship out of the gate with 0.10.0.

File: docs/site/content/docs/latest/ref-unmanaged-cluster.md
Comment (joshrosso) [reply docs/site/content/docs/latest/ref-unmanaged-cluster.md:113]: yup, still need to write it :+1:

File: docs/site/content/docs/latest/ref-unmanaged-cluster.md
Comment (joshrosso) [reply docs/site/content/docs/latest/ref-unmanaged-cluster.md:64]: yup, still need to write it :+1:

File: docs/site/content/docs/assets/install-package-to-cluster.md
Comment (joshrosso) [reply docs/site/content/docs/assets/install-package-to-cluster.md:31]: We won't be using the 0.10.0 package repo out of the gate; this won't be the initial output.

File: docs/site/content/docs/assets/install-package-to-cluster.md
Comment (joshrosso) [reply docs/site/content/docs/assets/install-package-to-cluster.md:35]: We won't be using the 0.10.0 package repo out of the gate; this won't be the initial output.

**Comment by joshrosso** [comment]
@jpmcb @kcoriordan 

Would be good to get one more review on this.

Namely, I've introduced a reference section[0] (which Kay has seen but I don't think John has).

[0]:

image

File: docs/site/content/docs/latest/ref-unmanaged-cluster.md
Comment (kcoriordan) [inline docs/site/content/docs/latest/ref-unmanaged-cluster.md:5]:
```suggestion
This is the reference documentation for the `unmanaged-cluster` plugin. If
this is your first time deploying an unmanaged cluster, see
[Getting Started with Unmanaged Clusters](getting-started-unmanaged/).
```

File: docs/site/content/docs/latest/ref-unmanaged-cluster.md
Comment (kcoriordan) [inline docs/site/content/docs/latest/ref-unmanaged-cluster.md:147]:
```suggestion
When creating unmanaged clusters, you can point to a different TKr using the `--tkr` flag.
TKrs for unmanaged clusters are available here:
```

File: docs/site/content/docs/latest/ref-unmanaged-cluster.md
Comment (kcoriordan) [inline docs/site/content/docs/latest/ref-unmanaged-cluster.md:141]:
```suggestion
Unmanaged clusters gather details on how to create a cluster from a Tanzu
```

File: docs/site/content/docs/latest/ref-unmanaged-cluster.md
Comment (kcoriordan) [inline docs/site/content/docs/latest/ref-unmanaged-cluster.md:150]:
```suggestion
`projects.registry.vmware.com/tce/tkr`. Query available TKrs using 
[imgpkg](https://carvel.dev/imgpkg), for example, 

```

File: docs/site/content/docs/latest/ref-unmanaged-cluster.md
Comment (jpmcb) [inline docs/site/content/docs/latest/ref-unmanaged-cluster.md:51]: very small nit: since this is a _directory_ should this be:

```diff
+ ~/.config/tanzu/tkg/unmanaged/${CLUSTER_NAME}/
- ~/.config/tanzu/tkg/unmanaged/${CLUSTER_NAME}
```

**Review by jpmcb — DISMISSED** [review]
🛑 Overall, looks awesome. Blocking until `main` is ready to take these docs changes

_Or_, we could point this PR to the `next` branch and that shouldn't deploy to the live Netlify site?

File: docs/site/content/docs/latest/ref-unmanaged-cluster.md
Comment (joshrosso) [inline docs/site/content/docs/latest/ref-unmanaged-cluster.md:61]: note: add details on how to configure/set configuration as a whole.

File: docs/site/content/docs/latest/ref-unmanaged-cluster.md
Comment (joshrosso) [inline docs/site/content/docs/latest/ref-unmanaged-cluster.md:1]: TODO(@joshrosso): Add content in for switching contexts between clusters.

File: docs/site/content/docs/latest/ref-unmanaged-cluster.md
Comment (kcoriordan) [inline docs/site/content/docs/latest/ref-unmanaged-cluster.md:143]:
```suggestion
[default TKr is
```

File: docs/site/content/docs/latest/ref-unmanaged-cluster.md
Comment (kcoriordan) [inline docs/site/content/docs/latest/ref-unmanaged-cluster.md:146]:
```suggestion
When creating clusters, you can point to a different TKr using the `--tkr` flag.
```

File: docs/site/content/docs/latest/ref-unmanaged-cluster.md
Comment (kcoriordan) [inline docs/site/content/docs/latest/ref-unmanaged-cluster.md:147]:
```suggestion
The TCE project keeps TKrs for unmanaged clusters at
```

File: docs/site/content/docs/latest/ref-unmanaged-cluster.md
Comment (kcoriordan) [inline docs/site/content/docs/latest/ref-unmanaged-cluster.md:182]:
```suggestion
You can then edit the above TKr in the `tkr/tkr-bom-v1.22.2.yaml`. After
```

File: docs/site/content/docs/latest/ref-unmanaged-cluster.md
Comment (joshrosso) [reply docs/site/content/docs/latest/ref-unmanaged-cluster.md:120]: Gonna keep these as upper level for the time being, I don't feel like they are sub topics.

File: docs/site/content/docs/latest/ref-unmanaged-cluster.md
Comment (joshrosso) [reply docs/site/content/docs/latest/ref-unmanaged-cluster.md:139]: Gonna keep these as upper level for the time being, I don't feel like they are sub topics.

File: docs/site/content/docs/latest/ref-unmanaged-cluster.md
Comment (joshrosso) [reply docs/site/content/docs/latest/ref-unmanaged-cluster.md:1]: going to keep this out of the docs until we implement a context-switch capability.

File: docs/site/content/docs/assets/unmanaged-desc.md
Comment (kcoriordan) [inline docs/site/content/docs/assets/unmanaged-desc.md:3]: this link is broken 
```suggestion
our guidance around [managed-clusters](../getting-started).
```

**Review by davidvonthenen — APPROVED** [review]
approving for the release. all feedback has been addressed

**Pull request merged**
